monitor/src/main/java/dk/dbc/holdingsitems/monitor/MonitorApplication.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (C) 2018 DBC A/S (http://dbc.dk/)
+ *
+ * This is part of holdings-items
+ *
+ * holdings-items is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * holdings-items is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package dk.dbc.holdingsitems.monitor;
+
+import javax.ws.rs.ApplicationPath;
+import javax.ws.rs.core.Application;
+
+/**
+ *
+ * @author DBC {@literal <dbc.dk>}
+ */
+@ApplicationPath("/api")
+public class MonitorApplication extends Application {
+}
```

Comment:
Status klassen skal ikke registreres? Mangler der ikke en `@Override getClasses()`?

Reply:
Det er ikke nødvendigt... Jeg ved godt JP er glad for det, men da typen er ejb (@Singleton/@Stateless/@Statefull) bliver de autoregistreret.